app/views/tutors/show.html.erb

```diff
@@ -358,6 +358,17 @@
 
     <hr>
 
+    <!-- ================================================================= -->
+    <div class="row">
+      <h4 class="col-md-12"><strong>Affiliate</strong></h4>
+    </div>
+    <div class="row">
+      <div class="col-md-4">
+        <strong>Affiliate:</strong><br>
+        <%= Affiliate.where(affiliate_id: @tutor.affiliate_id).name %>
```

Comment:
`.where` expects a hash. You're looking for `.find` which expects an integer id.